### DISCUSSION TITLE
[qterminal] How to reset the shortcut?

### DISCUSSION THREAD

**Discussion by gqfgh** [lxqt/lxqt#2099] [Q&A] [answered]
“Ctrl+C” is used by me to another shortcut, I couldn't stop the running command. Could I reset it?

**Comment by tsujan** [comment, marked as answer]
If you meant how to clear a shortcut, just press a modifier key (like Shift) when the shortcut editor has focus -- it's explained in the tooltips of Terminal settings → Shortcuts.

**Reply by gqfgh** [reply]
It works so well, thank you for your marvellous solution. (/≧▽≦)/~┴┴
